lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -589,6 +589,9 @@ may then be further optimized by selecting better implementations of each operat
         (with expr
             // Remove the select and struct node from the `expr` sum type, which will be replaced below.
             (exclude select struct)
+              
+            // Call Aggs either become calls to built-in-functions are aggregate functions
```

Comment:
```suggestion
            // A call_agg node represents either a collection aggregation function or an aggregation function of the  grouping operator
```

Reply:
This is strange actually. Why would we conflict these two? Any of the collection aggregation functions like "COLL_" should be represented by the normal function IR. By representing both of these things by `call_agg` requires splitting them later. It's over-complicating things